opendevin/core/metrics.py

```diff
@@ -0,0 +1,54 @@
+class Metrics:
+    """
+    Metrics class can record various metrics during running and evaluation.
+    Currently we define the following metrics:
+        total_cost: the total cost of the current LLM.
```

Comment:
we could say the unit is in USD.

Reply:
I think we should call it `accumulated_cost` rather than `total_cost`. It only records the cost so far, not the total cost for the entire task.